.storybook/preview-head.html

```diff
@@ -7,4 +7,26 @@
   body {
     background-color: initial !important;
   }
+
+  /**
+   * Percy-specific CSS doesn't play nicely with css-in-js solutions, so we
+   * need to add overrides here instead of in the relevant css files.
+   * See https://docs.percy.io/docs/percy-specific-css
+   *
+   * Selectors need to be in the format `[class^='thing'] because the actual
+   * class name will have a unique suffix attached to it.
```

Comment:
For people with little knowledge around this, perhaps an example could help?

```suggestion
   * class name (for example: `thing_20YWn`) will have a unique suffix attached to it.
```